clinica/pipelines/machine_learning_spatial_svm/spatial_svm_utils.py

```diff
@@ -1082,9 +1081,7 @@ def heat_solver_equation(input_image, g, FWHM, t_step, dartel_input):
     input_image_read = nib.load(input_image)
     input_image_data = input_image_read.get_fdata(dtype="float32")
 
-    u = utils.heat_solver_tensor_3D_P1_grad_conj(
-        input_image_data, g, beta, h, t_step, CL_value=None, epsilon=None
-    )
+    u = utils.heat_solver_tensor_3D_P1_grad_conj(input_image_data, g, beta, h, t_step)
```

Comment:
Why are we removing the default values here?

Reply:
Because this call will not trigger the default values for `CL_value` and `epsilon`. If we are not going to pass proper values to them, we should not specify them.